### PR TITLE
hw: bsp: frdm-k82f: update LED definitions

### PR DESCRIPTION
**Pull request by utzig**
Use standard LED_n nomenclature from color name based.

### CODE REVIEW

File: hw/bsp/frdm-k82f/include/bsp/bsp.h [hunk 1]
```diff
@@ -33,7 +33,9 @@ extern uint8_t __DATA_ROM;
 #define LED_RED_PIN         MCU_GPIO_PORTC(8)
 #define LED_GREEN_PIN       MCU_GPIO_PORTC(9)
 #define LED_BLUE_PIN        MCU_GPIO_PORTC(10)
+
 #define LED_BLINK_PIN       LED_RED_PIN
+#define LED_2               LED_GREEN_PIN
```
Comment (kasjer) [inline hw/bsp/frdm-k82f/include/bsp/bsp.h:38]: What happened to LED_1? :frowning:

Comment (utzig) [reply hw/bsp/frdm-k82f/include/bsp/bsp.h:38]: That is not used anywhere afaik, I am not sure why BSPs export it but I'll do it here too.

Comment (utzig) [reply hw/bsp/frdm-k82f/include/bsp/bsp.h]: I switched to using the standard LED_n nomenclature, thanks for the suggestion.

**Review by kasjer — APPROVED** [review]
I was just wandering why start with 2 and skip 1 and 3.
I did not mind color coded LEDs.

**Pull request merged**